### DISCUSSION TITLE
Documentation seems to be missing

### DISCUSSION THREAD

**Discussion by jollydodo** [thuliteio/doks#1124] [General]
I like this theme a lot and have some experience with Hugo so it was a breeze to get this working on my local machine. However, I noticed that there is barely any documentation available (most links to guides are dead). Is this is a known problem or are the old docs not relevant anymore because of some updates? Both Hyas and Doks don't feature any information anymore on deploying for example. Similarly, simple stuff like using some shortcodes for callouts / tab lists is all incorporated in this beautiful theme, but no documentation was available (which then took me some time to dig through the code to find out about these features).

So I'm wondering if some documentation got lost or if it never existed? (just so I know before I'm doing double work) ;)

**Comment by h-enk** [comment]
Nice to hear that you like this theme a lot. The restructured Doks theme and underlying Hyas framework now (as of today) have a solid code base to extend/improve upon. Now my first priority is to have a proper documentation base. I started working on that today 😆

**Reply by jollydodo** [reply]
Thank you for the information! I'll patiently await the documentation and keep messing around in the meantime ;). (I'm using this theme to try and create a new website for a Fablab in the Netherlands). Most of the stuff I already got working,  but things like adding the 3 most recent articles on the home page not (https://github.com/gethyas/doks/discussions/205 which was already discussed here, but that link is not working anymore which is the reason I got confused. But I now understand that this has to do with the complete restructure).

**Reply by h-enk** [reply]
Does this help you? https://github.com/gethyas/getdoks.org/blob/v0.4.2/layouts/index.html#L148

**Reply by jollydodo** [reply]
Worked like a charm :) Thanks!
